meson.build

```diff
@@ -67,11 +67,17 @@ if host_machine.endian() == 'big'
 endif
 
 p11_kit = dependency('p11-kit-1', required: false)
-if p11_kit.found()
+default_pkcs11_module = get_option('default_pkcs11_module')
+if default_pkcs11_module == 'no' and p11_kit.found()
   default_pkcs11_module = p11_kit.get_variable(pkgconfig: 'proxy_module')
-  conf.set_quoted('DEFAULT_PKCS11_MODULE', default_pkcs11_module)
 endif
 
+if default_pkcs11_module == ''
```

Comment:
This will now fail to catch that default_pkcs11_module is 'no' but p11_kit was not found to change it.